clients/drcachesim/tracer/tracer.cpp

```diff
@@ -1398,6 +1409,100 @@ event_delay_app_instruction(void *drcontext, void *tag, instrlist_t *bb, instr_t
     return DR_EMIT_DEFAULT;
 }
 
+
+typedef struct _func_metadata {
+    const trace_marker_type_t marker_retaddr;
+    const trace_marker_type_t marker_arg;
+    const trace_marker_type_t marker_retval;
+    const char *name;
+    const int num_args;
+} func_metadata;
+
+#define MALLOC_RETADDR TRACE_MARKER_TYPE_FUNC_MALLOC_RETADDR
```

Comment:
Not sure you gain much from these, other than saving a few keystrokes. I'd remove them.

Reply:
Done